packages/patternfly-4/react-core/src/components/AboutModal/AboutModal.d.ts

```diff
@@ -8,8 +8,7 @@ export interface AboutModalProps extends HTMLProps<HTMLDivElement> {
   trademark?: string;
   brandImageSrc: string;
   brandImageAlt: string;
-  logoImageSrc?: string;
-  logoImageAlt?: string;
+  bgImgSrc?: string;
```

Comment:
@seanforyou23 Can we call this backgroundImageSrc instead? Also do we need a backgroundImageAlt?

Reply:
Don't need an alt when it's just background styling